src/helm/benchmark/model_deployment_registry.py

```diff
@@ -27,68 +35,90 @@ class ModelDeployment:
 
     A model can have multiple model deployments."""
 
+    # Name of the model deployment.
+    # Usually formatted as "<hosting_group>/<engine_name>"
+    # Example: "huggingface/t5-11b"
```

Comment:
The old commenting style was better because VS Code and other IDEs will be able to extract and display them as tooltips.